includes/general.php

```diff
@@ -2423,6 +2426,7 @@ function pods_no_conflict_on( $object_type = 'post', $object = null ) {
 				$no_conflict['filter'] = array_merge( $no_conflict['filter'], array(
 					array( 'add_user_metadata', array( PodsInit::$meta, 'add_user_meta' ), 10, 5 ),
 					array( 'update_user_metadata', array( PodsInit::$meta, 'update_user_meta' ), 10, 5 ),
+					array( 'update_user_metadata_by_id', array( PodsInit::$meta, 'update_user_meta_by_id' ), 10, 4 ),
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 29fece6c39e82f32d02ee8d9a480312b</sub>